bin/plasma-chain-benchmark.js

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env node
+const genAccounts = require('../test/mock-accounts').genAccounts
+const Benchmark = require('benchmark')
+const logUpdate = require('log-update')
+const program = require('commander')
+const colors = require('colors') // eslint-disable-line no-unused-vars
+const MockNode = require('../src/mock-node.js')
+const axios = require('axios')
+const constants = require('../src/constants.js')
+const UnsignedTransaction = require('plasma-utils').serialization.models.UnsignedTransaction
+const BN = require('web3').utils.BN
+const Web3 = require('web3')
+const cluster = require('cluster');
+const numCPUs = require('os').cpus().length;
+
+const http = axios.create({
```

Comment:
Some minor linting stuff in this file, but I just went ahead and [opened a PR](https://github.com/plasma-group/plasma-chain-operator/pull/65) to ensure that linting errors get caught automatically. Nothing to worry about.